### ISSUE TITLE
How does applyImpulse work?

### ISSUE THREAD

**Issue by DelvarWorld** [schteppe/cannon.js#272]
See https://github.com/schteppe/cannon.js/blob/master/src/objects/Body.js#L750-L758

> Apply impulse to a **world** point.

but then

> Body.prototype.applyImpulse = function(impulse, **relative**Point){

**Comment by schteppe** [comment]
You're right, formulation is a bit bad. Wanna make a PR? In any case, thanks for spotting this.
